pcmpl-args.el

```diff
@@ -3129,10 +3129,15 @@ options found in its man page."
       (pcmpl-args-process-file "git" "help" "-a")
       (goto-char (point-min))
       (let ((cmds (copy-sequence pcmpl-args-git-commands)))
-        (while (re-search-forward "^[ ]+[[:alpha:]].*$" nil t)
-          (dolist (cmd (split-string (match-string 0)))
+        (while (re-search-forward
+                "^[	 ]+\\([^[:space:]]+\\)[[:space:]]+\\(.*?\\)$"
```

Comment:
Whitespaces unreadable, `\t\s` can be read without whitespace-mode (e.g. on Github). In `.*?` final `?` has no sense.

Reply:
Also suggested regexp will match output like this:
```
 foo
 bar
```
because newline is also `[:space:]`.
So the correct regexp would be `"^[\t\s]+\\([^[:space:]]+\\)[\t\s]+\\(.*\\)$"`.

Reply:
It is intended to match
```
 hello
 world
```

Reply:
Just for history: it's intended to match hello and world as two separate matches rather than one with two groups.